warehouse/query-core/src/main/java/datawave/query/tables/shard/CountAggregatingIterator.java

```diff
@@ -62,12 +62,7 @@ public boolean hasNext() {
                 ResultCountingIterator.ResultCountTuple tuple = unpackValue(entry.getValue());
 
                 // Merge the columnVisibilities
-                try {
-                    this.columnVisibilities.add(tuple.getVisibility());
```

Comment:
no parsing is actually done here.... in fact no exception could really be thrown here.